### PR TITLE
Git utils

### PR DESCRIPTION
**Pull request by jessicastewart-udemy**

### CODE REVIEW

File: internal/git/git.go [hunk 1]
```diff
@@ -1,27 +1,97 @@
...
+	fmt.Printf("\x1b[31;1m%s\x1b[0m\n", fmt.Sprintf("error: %s", err))
+	os.Exit(1)
+}
+
+// Info should be used to describe the example commands that are about to run.
+func Info(format string, args ...interface{}) {
+	fmt.Printf("\x1b[34;1m%s\x1b[0m\n", fmt.Sprintf(format, args...))
+}
+
+func clone(gitURL string, dir string) {
```
Comment (jshiv) [inline internal/git/git.go]: functions need a capital letter to be "public".
clone -> Clone
commit -> Commit

File: internal/git/git.go [hunk 2]
```diff
@@ -1,27 +1,97 @@
-package git
+package main
```
Comment (jshiv) [inline internal/git/git.go]: Change back to `git` for use as a module.
Did you change this for testing with `func main()`?

Comment (jessicastewart-udemy) [reply internal/git/git.go]: Yea just for testing :)

File: internal/git/git.go (hunk 1 above)
Comment (jessicastewart-udemy) [reply internal/git/git.go]: Thanks! I did not know. Comment addressed

**Pull request merged**